src/frontend/overlay/src/components/organisms/holder/ContestantViewHolder.tsx

```diff
@@ -28,6 +29,42 @@ export const VideoWrapper = styled.video`
   border-radius: ${c.GLOBAL_BORDER_RADIUS};
 `;
 
+export const TeamM2tsVideoWrapper = ({ url, setIsLoaded }) => {
+    const dispatch = useAppDispatch();
+    const videoRef = useRef<HTMLVideoElement>();
+    useEffect(() => {
+        setIsLoaded(false);
+        if (videoRef.current) {
+            const player = mpegts.createPlayer({
+                type: "mpegts",
+                isLive: true,
+                url: url,
+            });
```

Comment:
According to [this](https://github.com/xqq/mpegts.js/blob/master/docs/api.md#config), request header for video's auth can be added by config here.